app/components/TimeSeriesChart.tsx

```diff
@@ -125,13 +125,26 @@ export default function TimeSeriesChart({
   maxValue,
   unit,
 }: Props) {
+  // We use the largest data point +20% for the graph scale
+  // clamping at `maxValue` (if set) which is usually overall capacity
+  const calculatedMaxValue = useMemo(() => {
+    if (!maxValue) return null
```

Comment:
Don't you still want a reasonable max on the chart (like `dataMax * 1.2`) if none is specified from outside?